exercises/food-chain/example.js

```diff
@@ -29,51 +29,64 @@ FoodChain.prototype.verses = function (first, last) {
-      'I know an old lady who swallowed a horse.\n' + 'She\'s dead, of course!\n';
+  case 1:
+    verse = 'I know an old lady who swallowed a fly.\n' + 'I don\'t know why she swallowed the fly. Perhaps she\'ll die.\n';
+    break;
+
+  case 2:
+    verse = 'I know an old lady who swallowed a spider.\n' + 'It wriggled and jiggled and tickled inside her.\n' +
+            'She swallowed the spider to catch the fly.\n' + 'I don\'t know why she swallowed the fly. Perhaps she\'ll die.\n';
+    break;
+
+  case 3:
+    verse = 'I know an old lady who swallowed a bird.\n' + 'How absurd to swallow a bird!\n' +
+            'She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her.\n' +
+            'She swallowed the spider to catch the fly.\n' + 'I don\'t know why she swallowed the fly. Perhaps she\'ll die.\n';
+    break;
+
+  case 4:
+    verse = 'I know an old lady who swallowed a cat.\n' + 'Imagine that, to swallow a cat!\n' + 'She swallowed the cat to catch the bird.\n' +
+            'She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her.\n' + 'She swallowed the spider to catch the fly.\n' +
+            'I don\'t know why she swallowed the fly. Perhaps she\'ll die.\n';
+    break;
+
+  case 5:
+    verse = 'I know an old lady who swallowed a dog.\n' + 'What a hog, to swallow a dog!\n' + 'She swallowed the dog to catch the cat.\n' +
+            'She swallowed the cat to catch the bird.\n' + 'She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her.\n' +
+            'She swallowed the spider to catch the fly.\n' + 'I don\'t know why she swallowed the fly. Perhaps she\'ll die.\n';
+    break;
+
+  case 6:
+    verse = 'I know an old lady who swallowed a goat.\n' + 'Just opened her throat and swallowed a goat!\n' + 'She swallowed the goat to catch the dog.\n' +
+            'She swallowed the dog to catch the cat.\n' + 'She swallowed the cat to catch the bird.\n' +
+            'She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her.\n' +
+            'She swallowed the spider to catch the fly.\n' + 'I don\'t know why she swallowed the fly. Perhaps she\'ll die.\n';
+    break;
+
+  case 7:
+    verse = 'I know an old lady who swallowed a cow.\n' + 'I don\'t know how she swallowed a cow!\n' + 'She swallowed the cow to catch the goat.\n' +
+            'She swallowed the goat to catch the dog.\n' + 'She swallowed the dog to catch the cat.\n' + 'She swallowed the cat to catch the bird.\n' +
+            'She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her.\n' + 'She swallowed the spider to catch the fly.\n' +
+            'I don\'t know why she swallowed the fly. Perhaps she\'ll die.\n';
+    break;
+
+  case 8: verse = 'I know an old lady who swallowed a horse.\n' + 'She\'s dead, of course!\n';
+    break;
+
+  default: verse = '';
   }
+
+  return verse;
```

Comment:
I think we can remove the variable `verse` and return from each case. That would remove the multiple `break`s.

Reply:
I used to dislike having multiple `returns` in a function.  I have changed my mind about that though, especially in a structure like this where the semantics are really clear.

Also interesting to note is that I've run some cyclomatic complexity analyzers on code like this, and as far as they're concerned, ` if x 10` with one `return` is the same complexity as having 10 `return`s in one function.

Reply:
The previous version had as many returns as `case`s.

I was happy with that version (I don't remember who was the author), but ESLint didn't think like that. ESLint *forced* me to add a `default` case and a return value. That's why I created a local variable.

What I would discourage is to mix them: I want all `case`s (even the `default`) equal. All or none shoudl return value. To fix ESLint errors, it looked betteer to me to return just at the end of the method.

Reply:
Hmm.  I think this is a case where I don't exactly agree with the linting rules, but I don't feel strongly enough about that to suggest we make an exception for this exercise.

I am back to my original position: merge as is!

Reply:
I feel exactly like that. I don't like the rules in this case, but they are the rules we chose. So, it's some code I need to bear with. I'm not the code, so I need to understand that sometimes, it's good not to agree completely on how the code looks like if it's better for the team and for the long run.